src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -332,6 +333,18 @@ protected function pExpr_Array(Array_ $array): string
         return parent::pExpr_Array($array);
     }
 
+    protected function pExpr_BinaryOp_Pipe(Pipe $node, int $precedence, int $lhsPrecedence): string
+    {
+        return $this->pInfixOp(
+            Pipe::class,
+            $node->left,
+            $this->nl . $this->resolveIndentSpaces() . '|> ',
```

Comment:
if `resolveIndentSpaces()`, the normal `"\n"` should be used instead over `$this->nl`, I will create new PR for inside function/classmethod usage test